src/main.js

```diff
@@ -105,11 +104,8 @@ async function run() {
             )
             const registry =
               stateRepo.registry || defaultRegistries[dispatch.type]
-            const imageRepo = `${registry}/${originRepo}`
+            const imageRepo = `${registry}/${github.context.repo.owner}/${github.context.repo.repo}`
```

Comment:
The image repository could useful to be defined in the `make_dispatches.yml` config file right?

Something like:

```yaml
        registry: my_registry_address # OPTIONAL: if not specified uses org SNAPSHOTS or org RELEASES registry)
        image_repository: my-image #OPTIONAL: default to ${{ github.repository }}
```